build.sbt

```diff
@@ -242,9 +243,11 @@ lazy val algebirdBijection = module("bijection").settings(
 ).dependsOn(algebirdCore, algebirdTest % "test->test")
 
 lazy val algebirdSpark = module("spark").settings(
-    libraryDependencies += "org.apache.spark" %% "spark-core" % "1.3.0" % "provided",
-    crossScalaVersions := crossScalaVersions.value.filterNot(_.startsWith("2.12"))
-  ).dependsOn(algebirdCore, algebirdTest % "test->test")
+    libraryDependencies += "org.apache.spark" %% "spark-core" % sparkVersion % "provided",
+    crossScalaVersions := crossScalaVersions.value.filterNot(isScala210x),
+    mimaPreviousArtifacts := Set.empty,
+    scalacOptions := Seq.empty // Disable optimizations for now: https://github.com/scala/bug/issues/11247
```

Comment:
That's a bit extreme, since it'll also turn off other flags like -deprecation. I suggest doing something like this instead:
```scala
scalacOptions := scalacOptions.value.filterNot(_.contains("inline"))
```

Reply:
yep, thanks!